src/transformers/models/gpt2/modeling_gpt2.py

```diff
@@ -395,9 +471,20 @@ def _init_weights(self, module):
             module.bias.data.zero_()
             module.weight.data.fill_(1.0)
 
-    def _set_gradient_checkpointing(self, module, value=False):
-        if isinstance(module, GPT2Model):
-            module.gradient_checkpointing = value
+        def _set_gradient_checkpointing(self, module, value=False):
+            if isinstance(module, GPT2Model):
+                module.gradient_checkpointing = value
```

Comment:
This is one indentation level too deep. There shouldn't be any modification to that method.

Reply:
Yes - this is totally my bad, just fixed this.

Reply:
No problem, wanted to make sure you saw it :-)